x-pack/plugins/timelines/public/components/fields_browser/index.tsx

```diff
@@ -20,22 +18,15 @@ export type {
 
 const EMPTY_BROWSER_FIELDS = {};
 
-export type FieldBrowserWrappedComponentProps = FieldBrowserProps & {
-  store: Store;
-};
-
-export const FieldBrowserWrappedComponent = (props: FieldBrowserWrappedComponentProps) => {
-  const { store, ...restProps } = props;
+export const FieldBrowserWrappedComponent = (props: FieldBrowserProps) => {
```

Comment:
This component was used by `x-pack/plugins/timelines/public/methods/index.tsx`,
but found it deleted in this change.
Is this component still in use anywhere else?